src/lib/radio/radio.ts

```diff
@@ -280,10 +300,26 @@ export class MdRadioButton implements OnInit {
   name: string;
 
   /** Used to set the 'aria-label' attribute on the underlying input element. */
-  @Input('aria-label') ariaLabel: string;
+  _ariaLabel: string;
+  @Input('aria-label')
+  get ariaLabel(): string {
+    return this._ariaLabel;
+  }
+  set ariaLabel(value: string) {
+    this._ariaLabel = value;
+    this._change.markForCheck();
```

Comment:
For `labelPosition` and `disabled`, group buttons' values depend on group's values. When the group's `disabled` changed, the button's `get disabled` also changed but `set disabled` is not called and button's `_disabled` is not changed. We need to let buttons `markForCheck()` when the group value changed. I added a `_groupValueChanged()` function but not sure it's the right way to do.